### PR TITLE
Update footer content

### PR DESCRIPTION
**Pull request by toddbaert**
- add linkedin to community (footer)
- add link to community repo (footer)
- add link to openfeature.dev (footer)
- fix footer column alignment (not center justified)
- swizzles footer to add CNCF logo

Closes: https://github.com/open-feature/docs.openfeature.dev/issues/69

### CODE REVIEW

File: src/theme/Footer/Layout/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      className={clsx('footer', {
+        'footer--dark': style === 'dark',
+      })}
+    >
+      <div className="container container-fluid">
+        {links}
+        {(logo || copyright) && (
+          <div className="footer__bottom text--center">
+            {logo && <div className="margin-bottom--sm">{logo}</div>}
+            {isDarkTheme ? <CncfIconDark style={{ width: 300 }} /> : <CncfIconDLight style={{ width: 300 }} />}
```
Comment (beeme1mr) [inline src/theme/Footer/Layout/index.tsx:24]: We only need the light version of the logo since it's in the footer.

File: docusaurus.config.js [hunk 2]
```diff
@@ -173,7 +165,7 @@ const config = {
             ],
           },
         ],
-        copyright: `© ${new Date().getFullYear()} OpenFeature | Documentation Distributed under CC BY 4.0 | All Rights Reserved`,
+        copyright: `© ${new Date().getFullYear()} OpenFeature is a Cloud Native Computing Foundation sandbox project | Documentation Distributed under CC BY 4.0 | All Rights Reserved`,
```
Comment (beeme1mr) [inline docusaurus.config.js:168]: Can we put this on two lines now? It's getting pretty long.

File: docusaurus.config.js [hunk 3]
```diff
@@ -150,6 +130,18 @@ const config = {
                 label: 'Twitter',
                 href: 'https://twitter.com/openfeature',
               },
+              {
+                label: 'LinkedIn',
+                href: 'https://www.linkedin.com/company/openfeature/',
+              },
+              {
```
Comment (beeme1mr) [inline docusaurus.config.js:137]: The website could be moved under "more"

File: docusaurus.config.js [hunk 4]
```diff
@@ -150,6 +130,18 @@ const config = {
...
+                label: 'Website',
+                href: 'https://openfeature.dev/'
+              },
+              {
+                label: 'Community',
```
Comment (beeme1mr) [inline docusaurus.config.js:142]: Perhaps this should say something like "Join the Community". It feels weird to have a community item under the community column.

File: src/theme/Footer/Layout/index.tsx (hunk 1 above)
Comment (toddbaert) [reply src/theme/Footer/Layout/index.tsx:24]: I think you're right.

File: docusaurus.config.js (hunk 2 above)
Comment (toddbaert) [reply docusaurus.config.js:168]: The text wraps automatically.

Comment (beeme1mr) [reply docusaurus.config.js:168]: Okay, let's leave it for now.

**Review by beeme1mr — APPROVED** [review]

**Pull request merged**